### DISCUSSION TITLE
Docker NewPipe

### DISCUSSION THREAD

**Discussion by Lowitle** [TeamNewPipe/NewPipe#11739] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Possibility of making a docker NewPipe

### Additional information

I'm thinking that it would be interesting to have our newpipes accessible anywhere (using a login) in a docker machine so we can access it through our browser too.

It would be interesting to have all our data in our server instead of our smartphone (that we can lose).

**Comment by opusforlife2** [comment, marked as answer]
https://piped.video
